src/pages/SingleVideo/SingleVideo.css

```diff
@@ -0,0 +1,27 @@
+.player {
+  width: 100%;
+  height: 50vh;
+  border-radius: var(--br);
+}
+
+.video-controls {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  padding: 0.5rem;
+  background-color: rgba(0, 0, 0, 0.2);
```

Comment:
As you have used color variables everywhere else.
so let's try to be more consistent in it